Makefile

```diff
@@ -2,8 +2,15 @@
 export GOBIN ?= $(shell pwd)/bin
 
 GOLINT = $(GOBIN)/golint
+GEN_ATOMICINT = $(GOBIN)/gen-atomicint
+GEN_VALUEWRAPPER = $(GOBIN)/gen-valuewrapper
 
-GO_FILES ?= *.go
+GO_FILES ?= $(shell find . '(' -path .git -o -path vendor ')' -prune -o -name '*.go' -print)
```

Comment:
what is `GO_FILES` used for?

Reply:
Currently, just `gofmt` checking. Although I realize now that it should be used for some of the other targets too. Fixing.